[sysdig]: KSPM bugs fix- add CPU limit, merge deploy flags and add condition to sa

yaminSapir opened the pull request:
## What this PR does / why we need it:
Bug fix:
* SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
* SSPROD-14088: KSPM components - Merge both flags into 1
* SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

yaminSapir committed:
SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
SSPROD-14088: KSPM components - Merge both flags into 1
SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed

yaminSapir committed:
SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
SSPROD-14088: KSPM components - Merge both flags into 1
SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed

yaminSapir committed:
SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
SSPROD-14088: KSPM components - Merge both flags into 1
SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed

yaminSapir committed:
SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
SSPROD-14088: KSPM components - Merge both flags into 1
SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed

yaminSapir committed:
SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
SSPROD-14088: KSPM components - Merge both flags into 1
SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed

yaminSapir committed:
SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
SSPROD-14088: KSPM components - Merge both flags into 1
SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed

yaminSapir committed:
Merge branch 'master' into SSPROD-14258-fix-kspm-charts

lilx1ao commented on charts/sysdig/CHANGELOG.md:
```diff
@@ -4,6 +4,12 @@
+### Minor changes
+* KSPM Analyzer - fix container missing CPU\Memory limits & requests
+* KSPM components - Merge analyzer and collector deploy flags into 1
+* KSPM Chart components - fix KSPM serviceAccount that is created when KSPM not installed
```
where are the changes with the first and the third items?

yaminSapir replied:
1 - daemonset-node-analyzer.yaml - added resources
3 - serviceaccount-kspm-collector.yaml - added a condition

lilx1ao replied:
ah I see

yaminSapir committed:
Merge branch 'master' into SSPROD-14258-fix-kspm-charts

yaminSapir committed:
SSPROD-14258: KSPM Analyzer - container missing CPU\Memory limits & requests
SSPROD-14088: KSPM components - Merge both flags into 1
SSPROD-14062: KSPM Chart components - KSPM serviceAccount is created when KSPM not installed
